wpimath/src/main/java/edu/wpi/first/math/StateSpaceUtil.java

```diff
@@ -206,8 +206,6 @@ public static Matrix<N4, N1> poseTo4dVector(Pose2d pose) {
   @Deprecated(forRemoval = true, since = "2025")
   public static Matrix<N3, N1> poseTo3dVector(Pose2d pose) {
     return VecBuilder.fill(
-        pose.getTranslation().getX(),
-        pose.getTranslation().getY(),
-        pose.getRotation().getRadians());
+        pose.getTranslation().y(), pose.getTranslation().x(), pose.getRotation().getRadians());
```

Comment:
```suggestion
        pose.getTranslation().x(), pose.getTranslation().y(), pose.getRotation().getRadians());
```
To be fair this is deprecated for removal, but still.